CONTRIBUTING.md

```diff
@@ -3,18 +3,18 @@
 For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
 
 - [Development](#development)
-  * [Sequence diagrams](#sequence-diagrams)
-  * [Running a Conjur server locally](#running-a-conjur-server-locally)
-  * [Running a Puppet master locally](#running-a-puppet-master-locally)
-  * [Running a Puppet node locally](#running-a-puppet-node-locally)
-  * [Generating the documentation](#generating-the-documentation)
+  + [Sequence diagrams](#sequence-diagrams)
+    - [Manifest-Provided Identity](#manifest-provided-identity)
+    - [Pre-Provisioning Using Host Identity](#pre-provisioning-using-host-identity)
+  + [Running a test environment](#running-a-test-environment)
+  + [Running a Puppet node locally](#running-a-puppet-node-locally)
 - [Testing](#testing)
+- [Generating the documentation](#generating-the-documentation)
 - [Releases](#releases)
-  * [Verify and update dependencies](#verify-and-update-dependencies)
-  * [Publishing a Release](#publishing-a-release)
-
-<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents
-generated with markdown-toc</a></i></small>
+  + [Verify and update dependencies](#verify-and-update-dependencies)
+  + [Publishing a Release](#publishing-a-release)
+- [Running a Puppet master and Windows-based Puppet node locally](#running-a-puppet-master-and-windows-based-puppet-node-locally)
```

Comment:
Should the `Running a Puppet master and Windows-based Puppet node locally` section be moved up in the document? I know it's sort of a special case, but it might be better suited to be just after the `Running a Puppet node locally` (maybe renaming that section to `Running a Linux Puppet node locally`?